main.go

```diff
@@ -34,7 +34,7 @@ func main() {
 	}
 
 	stopChan := make(chan string)
-	go master(stopChan)
+	go master(&stopChan)
```

Comment:
hmm. 🤔 I was doing this when I was debugging the proxy and closing channels. I think it's safe to revert this to not pass a pointer. My original understanding here is that we wanted to effect the actual `stopChan` we're creating on line 36 rather than a copy that's being passed into the `master` go routine. Am I using the wrong syntax here? Should it be `*stopChan`? or am I missing the point of how Channels / Pointers / Copies work?